Dockerfile.deploy

```diff
@@ -1,11 +1,6 @@
-FROM fabric8/fabric8-openshift-nginx:vd83b3a1
-MAINTAINER "Pete Muir <pmuir@bleepbleep.org.uk>"
+FROM nginx:latest
```

Comment:
Using :latest is bad practice and cause indeterministic behavior as what the image contains depends on when it was built causing irreproducible builds and you have given up control over which version you're using in your production env.